src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -0,0 +1,89 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+* Servlet class responsible for user registration.
+*/
+public class RegisterServlet extends HttpServlet {
+
+ @Override
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
```

Comment:
Not sure if this is marked green because the other commits from this branch haven't been merged yet, but if you are copy and pasting every time it might be more helpful to just copy in the new stuff so that its more obvious what has been changed (but I know it also might just be that I haven't done the other code reviews yet)

Reply:
Yeah there is a lot of green since we didnt merge the previous requests!